pmm/pmm2-server-release.groovy

```diff
@@ -5,22 +5,22 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
 
 pipeline {
     environment {
-        specName = 'pmm-release'
+        specName = 'pmm2-release'
     }
     agent {
         label 'master'
     }
     parameters {
         choice(
-            choices: 'laboratory\npmm-hotfix-laboratory',
-            description: 'publish pmm-server packages from regular(laboratory) or hotfix(pmm-hotfix-laboratory) repository',
+            choices: 'laboratory',
+            description: 'publish pmm2-server packages from regular(laboratory) repository',
             name: 'UPDATER_REPO')
         string(
-            defaultValue: 'perconalab/pmm-server:dev-latest',
+            defaultValue: 'perconalab/pmm-server:2',
```

Comment:
What default tag will we use for pmm-server docker?

Reply:
for pmm2 - dev-latest

Reply:
@EvgeniyPatlan  in this case 'perconalab/pmm-server:1' will be for pmm?